src/ray/gcs/gcs_server/gcs_task_manager.cc

```diff
@@ -322,6 +322,7 @@ GcsTaskManager::GcsTaskManagerStorage::AddOrReplaceTaskEvent(
 
     // Update iter.
     next_idx_to_overwrite_ = (next_idx_to_overwrite_ + 1) % max_num_task_events_;
+    total_num_events_ += 1;
```

Comment:
@rickyyx is this the right place to log number of tasks?

Reply:
I think we should log it here: https://github.com/ray-project/ray/blob/ed72ca8f10a09c9934a51fc3579d11c64fda58b4/src/ray/gcs/gcs_server/gcs_task_manager.cc#L261-L262

One thing to note is, since we always do evictions, we might have evicted a task before but we thought it was a new task, so there is the risk of undercounting. But this will only happen if we have >100K tasks, and really slow reporting. But I guess this is probably fine?

Reply:
yup i think that's probably fine..